webclient/dokument/urls.py

```diff
@@ -62,4 +62,9 @@
         post_ajax_get_3d_limit,
         name="post_ajax_get_3d_limit",
     ),
+    path(
```

Comment:
@jnihnat Předpokládám, že musím přidat i oprávnění do URLs ve shodě s oprávněním pro detail AZ. Draftnul jsem to zde: https://docs.google.com/spreadsheets/d/1it_kr3HCN2vv54SUvn696XaP9tKCgcHnqUCBeg4CGYw/edit?gid=0#gid=0 (řádek 116). Jen prosím o potvrzení, že nedělám něco blbě. Mám takový dojem, že tabulka se použije i u projektů (je to tak?), takže by to pak mělo být rozlišené pomocí odlišné action a opravnění být jednou vázáno na projekt a podruhé na AZ, ne?

Reply:
pridane: dokumenty_tabulka_projekt a dokumenty_tabulka_arch_z